content/manuals/build/debug/pprof.md

```diff
@@ -0,0 +1,108 @@
+---
+linkTitle: Profiling builds
+title: Profiling builds with pprof
+description: |
+  Learn how you can use `pprof` to analyze and debug your builds.
+---
+
+{{< introduced buildx 0.18.0 >}}
+
+You can configure Buildx to generate [`pprof`](https://github.com/google/pprof)
+memory and CPU profiles to analyze and optimize your builds. These profiles
```

Comment:
This won't really help anybody to optimize their builds. It's a profile of the Go code and not of their builds. And since builds are mostly performed on the server and this is a profile of the client, you likely wouldn't even see anything from your build on there even if it was in there.

Reply:
Thanks. I will delete this page then I think.

Reply:
We could move this page to buildx repo I think instead. Probably within https://github.com/docker/buildx/blob/master/.github/CONTRIBUTING.md

Reply:
good idea, raised this: docker/buildx#2757